rootfs-builder/README.md

```diff
@@ -0,0 +1,62 @@
+# Building a rootfs for Kata Containers Guest OS #
```

Comment:
"rootfs" needs defining, and we should probably define "OS" for completeness too. I'm wondering if it should also be O/S as that's conventional too. I'm sure @iphutch can tell us ;)